en/docs/chapter_computational_complexity/performance_evaluation.md

```diff
@@ -2,10 +2,10 @@
 
 In algorithm design, we pursue the following two objectives in sequence.
 
-1. **Finding a Solution to the Problem**: The algorithm should reliably find the correct solution within the stipulated range of inputs.
+1. **Finding a Solution to the Problem**: The algorithm should reliably find the correct solution within the specified range of inputs.
 2. **Seeking the Optimal Solution**: For the same problem, multiple solutions might exist, and we aim to find the most efficient algorithm possible.
 
-In other words, under the premise of being able to solve the problem, algorithm efficiency has become the main criterion for evaluating the merits of an algorithm, which includes the following two dimensions.
+In other words, under the premise of being able to solve the problem, algorithm efficiency has become the main criterion for evaluating the quality of an algorithm, which includes the following two dimensions.
```

Comment:
it might be better here to just not have anything, as in “for evaluating an algorithm”